src/style.css

```
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
tars--2 .stars__star:nth-child(-n+2),
... {
```
Consider grouping selectors with a comma to avoid code repeating